templates/index.html

```diff
@@ -78,6 +78,7 @@
             <a href="#main-content">Jump to main content</a>
           </span>
           <ul class="p-navigation__links" role="menu">
+            <li class="p-navigation__link" role="menuitem"><a class="p-link--external" href="/store/">Store</a></li>
```

Comment:
The external class should be on the build navigation item.

Reply:
done, silly me